features/rules/ALS/ALS005_Alignment-shape-representation.feature

```diff
@@ -14,24 +14,36 @@ The rule verifies that each IfcAlignment uses correct representation.
   @E00020
   Scenario: Agreement on each IfcAlignment using correct representation - Value
 
-    Then The value of attribute RepresentationIdentifier must be FootPrint or Axis
+    Then .RepresentationIdentifier. ^is^ 'FootPrint' or 'Axis'
 
 
   @E00010
-  Scenario: Agreement on each IfcAlignment using correct representation - Type = 'Axis'
+  Scenario: Agreement on each IfcAlignment using correct representation - horizontal only
 
     Given .RepresentationIdentifier. ^is^ 'Axis'
-    Then The value of attribute RepresentationType must be Curve3D
+    Given .RepresentationType. ^is^ 'Curve2D'
```

Comment:
This would probably also be nice to do for other files (e.g. ALS006 & ALS007), so we can get rid of another step implementation.